tests/framework/ROM/TimeSeries/ARMA/preserve_cdf.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/ROM/TimeSeries/ARMA.PreserveCdf</name>
+    <author>talbpaul</author>
+    <created>2019-01-22</created>
+    <classesTested>SupervisedLearning.ARMA</classesTested>
+    <description>
+      Tests the feature to preserve the CDF of the input training data when producing synthetic histories.
+      The standard is being able to identically match the original data's ValueDuration curve.
```

Comment:
I think it is worth to add the issue number here.

Reply:
I don't think that's standard practice. If you want that requirement, we should probably discuss it. It sounds like a generally okay idea to me, but perhaps not all regression tests originate with an issue, right? There's a specific tag for requirements-based tests, but this feature is not significant enough to be requirements-based.

Reply:
It is not general standard practice. What I mean is that the issue for your case have more detailed description regarding this feature. It is fine with me if you do not include the issue number in the description, but I think a little bit more descriptions will help future developers to understand this feature much quickly.

Reply:
After discussion, it became apparent that the desired verbosity was to assure the user guide will have access to the description in the issue for guidance on how to create synthetic histories. I will pass this information on to the creator of the user guide so they can make use of it.